modules/nw-ovn-k-adminnetwork-policy.adoc

```diff
@@ -0,0 +1,80 @@
+
+An `AdminNetworkPolicy` (ANP) is a cluster-scoped custom resource definition (CRD). As a {product-title} administrator, you can use ANP to secure your network by creating network policies before creating namespaces. Additionally, you can create network policies on a cluster-scoped level that is non-overridable by `NetworkPolicy` objects.
+
+The key difference between `AdminNetworkPolicy` and `NetworkPolicy` objects are that the former is for administrators and is cluster scoped while the latter is for tenant owners and is namespace scoped.
+
+An ANP allows administrators to specify the following:
+
+* A `priority` value that determines the order of its evaluation. The lower the value the higher the precedence.
+
+* A `subject` that consists of a set of pods that is selected by the policy.
+
+* A list of ingress rules to be applied for all ingress traffic towards the `subject`.
+
+* A list of egress rules to be applied for all egress traffic from the `subject`.
+
+[NOTE]
+====
+The `AdminNetworkPolicy` resource is a `TechnologyPreviewNoUpgrade` feature that can be enabled on test clusters that are not in production. The feature is behind the flag `enable-admin-network-policy`. For more information on feature gates and `TechnologyPreviewNoUpgrade` features, see "Enabling features using feature gates" in the "Additional resources" of this section.
+====
+
+[discrete]
+[id="adminnetworkpolicy-example_{context}"]
+== AdminNetworkPolicy example
+
+.Example YAML file for an ANP
+[%collapsible]
+====
+[source,yaml]
+----
+apiVersion: policy.networking.k8s.io/v1alpha1
+kind: AdminNetworkPolicy
+metadata:
+  name: sample-anp-deny-pass-rules <1>
+spec:
+  priority: 50 <2>
+  subject:
+    namespaces:
+      matchLabels:
+          kubernetes.io/metadata.name: example.name <3>
+  ingress: <4>
+  - name: "deny-all-ingress-tenant-1" <5>
+    action: "Deny"
+    from:
+    - pods:
+        namespaces:
+          namespaceSelector:
+            matchLabels:
+              custom-anp: tenant-1 <6>
+        podSelector:
+          matchLabels:
+            custom-anp: tenant-1 <7>
+  egress:<8>
+  - name: "pass-all-egress-to-tenant-1"
+    action: "Pass"
+    to:
+    - pods:
+        namespaces:
+          namespaceSelector:
+            matchLabels:
+              custom-anp: tenant-1
+        podSelector:
+          matchLabels:
+            custom-anp: tenant-1
+----
+<1> Specify a name for your ANP.
+<2> The `spec.priority` field supports a maximum of 100 ANP in the values of 0-99 in a cluster. The lower the value the higher the precedence. Creating `AdminNetworkPolicy` with the same priority creates a nondeterministic outcome.
+<3> Specify the namespace to apply the ANP resource.
+<4> ANP have both ingress and egress rules. ANP rules for `spec.ingress` field accepts values of `Pass`, `Deny`, and `Allow` for the `action` field.
+<5> Specify a name for the `ingress.name`.
+<6> Specify the namespaces to select the pods from to apply the ANP resource.
```

Comment:
Should the <6> be next to namespaces field instead of matchLabels.
Technically it could be matchLabels or matchExpressions for namespaceSelector.